src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+const Big = require('big.js');
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  const bigOne = new Big(numberOne);
+  const bigTwo = new Big(numberTwo);
+
+  if (operation === '+') {
+    result = bigOne.plus(bigTwo);
+  } else if (operation === '-') {
+    result = bigOne.minus(bigTwo);
+  } else if (operation === 'X') {
+    result = bigOne.times(bigTwo);
+  } else if (operation === '÷') {
+    result = bigOne.div(bigTwo);
```

Comment:
### [Required]
You've not managed division by ZERO...please handle this
![calc](https://user-images.githubusercontent.com/40712459/83511459-947e3380-a4c6-11ea-828c-7c540e127c5e.gif)

Reply:
As you can see from gif, in beginning, normal division wasn;t working, i dont know why....then i refreshed, works fine...except for DIVISION BY  ZERO

Also please try to increase the font size of buttons a little.....looks smaller and you got space, thanks